### PR TITLE
[improve]: Add additional instruction to readme

### PR DESCRIPTION
**Pull request by TRohit20**
Modified the current `readme.md` file by adding couple of additional instructions just to make it more clear and easier to setup Kubescape.io website locally making it easier to work with it and contribute if one wishes to. 

Thanks!

Screenshot 2023-09-19 at 11 04 30 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 site/
+myvenv
```
Comment (craigbox) [inline .gitignore:3]: This is custom to your own install and probably shouldn't be checked in.

File: README.md [hunk 2]
```diff
@@ -6,17 +6,19 @@ This repository contains the source files for the kubescape.io website.
 
 ### Local testing
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme to built the website.
```
Comment (craigbox) [inline README.md:9]: This is the readme for the website repo. The next line talks about building and testing, so I don't think we need to explain what these are for 🙂

File: README.md [hunk 3]
```diff
@@ -6,17 +6,19 @@ This repository contains the source files for the kubescape.io website.
...
-
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+* Ensure you have python and pip installed.
+* you can install all the required dependencies such as rrs plugin, mkdocs and more using the following command:
+```bash 
+pip install -r requirements.txt
+```
+* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker.
+* Once you have all the dependencies installed, run `mkdocs build` and the site will be built in the `site/` directory.
+* After building the website locally, you can run the wesbite locally using the command `mkdocs serve`.
```
Comment (craigbox) [inline README.md:20]:
````suggestion
* Ensure you have `python` and `pip` installed.
* Install the required dependencies:
```bash 
pip install -r requirements.txt
```
* Run `mkdocs serve`

## Local build

Run `mkdocs build`, and the site will be built in the `site/` directory.
````

Comment (craigbox) [reply README.md:20]: * running `pip` with the requirements file will install mkdocs-material, so we don't need to refer to that.
* Beware of "wesbite" vs "website".
* I suggest we keep testing and build instructions separate.

**Review by craigbox — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (TRohit20) [reply README.md:20]: Done, Resolved in new commit

**Pull request closed without merging**